exporters/otlp/otlplog/otlploggrpc/internal/transform/log_test.go

```diff
@@ -226,6 +227,81 @@ func TestResourceLogs(t *testing.T) {
 	assert.Equal(t, want, ResourceLogs(records))
 }
 
+func TestResourceLogsPerResource(t *testing.T) {
```

Comment:
@XSAM It definitely make sense and as @pellared also highlighted in the previous comment we should have better(strict) assertions for this scenario, so it will address that also. I have addressed this and made suggested changes.


@XSAM Happy to help :). Feel free to tag me wherever I can help, I am very active on consumer side of opentelemetry. I would love to contribute more and help in this amazing project that you all are maintaining.